pkg/workspace/config.go

```diff
@@ -0,0 +1,38 @@
+package workspace
+
+import (
+	"kusionstack.io/kusion/pkg/models"
+)
+
+// Config is used to describe the config of a workspace.
+type Config struct {
```

Comment:
Why not just use `type Workspace struct` here? What's the difference between `Config` and `Workspace`?